x/chainlink/exported/exported.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: MIT
+
+package exported
```

Comment:
just out of curiosity, what are these interfaces for?

Reply:
You have the ability to export some interface to allow developer of an Cosmos App to read your data easily
eg: https://github.com/cosmos/cosmos-sdk/blob/master/x/bank/exported/exported.go

Reply:
I see, we need to export the data in our module for another modules to use for sure. I assume that those commented out interfaces registered in `RegisterInterfaces` are also needed.

Reply:
Yep exact :)